capitalize_decorator.rb

```diff
@@ -0,0 +1,5 @@
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
```

Comment:
- [ ]  @BenMKT Great work adding the class to capitalize the name, however, you forgot to import the decorator class before the capitalize decorator class inherits from it.

Reply:
- Import Decorator to resolve bug